src/server/vault.py

```diff
@@ -109,6 +109,19 @@ def get_exportable_key(app):
         app.logger.error("Failed to get exportable key from vault: %s", err)
     return None
 
+def test_private_key_file(app) -> bool:
+    # If the private key is present, just return
+    if os.path.isfile('id_ecdsa'):
+        app.logger.info("Private ssh key file present")
+        return True
+    
+    # Private key is not present, try to fetch or regenerate it
+    app.logger.info("Private ssh key file not present - refetching...")
```

Comment:
Do you want to say something less definitive than 'refetching'? 'Attempting to fetch' makes it sound like the outcome is in doubt, which it is.

Reply:
Changed.